CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+##  (2022-09-06)
+
```

Comment:
@AkbaraliShaikh Lets's actually get rid of changelog file. We are managing the changelog on Github releases. This is duplication of effort and is manual.

Reply:
okay.